pkgs/applications/science/biology/samtools/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   pname = "samtools";
-  version = "1.10";
+  version = "1.11";
 
   src = fetchurl {
```

Comment:
Can you use fetchGithub please?

Reply:
Unfortunately not in this case. The automatic package generated by GitHub is incomplete as stated by the authors on the [releases page](https://github.com/samtools/samtools/releases).

Reply:
We are not using the bundled library. They generated files are used and would be pointless to regenerate, right?